CMakeLists.txt

```diff
@@ -20,18 +20,23 @@ set (CMAKE_PROJECT_VERSION_MAJOR "0")
 set (CMAKE_PROJECT_VERSION_MINOR "1")
 set (CMAKE_PROJECT_VERSION_PATCH "0")
 
+option (RUN_TEST
+  "If false, build libsxg without building tests.\
+  Building tests of libsxg requires more recent version of cmake (>= 3.13).\
+  Tests include fuzzer which is currently supported only in clang++, \
```

Comment:
I may simply say "Tests include fuzzer will be disabled automatically if it is not supported by the compiler." but it is your choice.